src/astro/databases/base.py

```diff
@@ -139,10 +135,11 @@ def drop_table(self, table: Table) -> None:
     # ---------------------------------------------------------
     def load_file_to_table(
         self,
-        source_file: str,
+        source_file: File,
         target_table: Table,
         if_exists: LoadExistStrategy = "replace",
         chunk_size: int = DEFAULT_CHUNK_SIZE,
+        normalize_config: Optional[dict] = None,
```

Comment:
I strongly recommend that `normalize_config` is an argument in `File` as opposed to the `load_file_to_table` operation. The reason is that this parameter is file-specific and doesn't make sense for many of the use-cases!

Reply:
Good catch, making the change.